distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc

```diff
@@ -0,0 +1,32 @@
+:title: Installing from ${command-console}
+:type: installing
+:status: published
+:summary: Installing from ${command-console}.
+:project: ${branding}
+:order: 08
+
+===== Completing Installation from the ${command-console}
+
+In order to install ${branding} from the ${command-console}, use the command `profile:install <profile-name>`.
+The `<profile-name>` should be the desired <<_setup_types, Setup Type>> in lowercase letters.
```

Comment:
:question: Should you mention that there is a profile:list command, so an admin can see all of the available profiles?

Reply:
I like this idea

Reply:
Addressed